_rules/SC4-1-2-aria-state-or-property-allowed.md

```diff
@@ -17,7 +17,7 @@ authors:
 
 ### Applicability
 
-Any [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is specified on an HTML or SVG element.
+Any [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is [included-in-the-accessibility-tree](#included-in-the-accessibility-tree) and that is specified on an HTML or SVG element.
```

Comment:
> Any [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is [included-in-the-accessibility-tree](#included-in-the-accessibility-tree) and that is specified on an HTML or SVG element.

-->

> Any [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is specified on an HTML or SVG element that is [included-in-the-accessibility-tree](#included-in-the-accessibility-tree).

... since it is the element that is included in the accessibility tree, not the state/property

Reply:
Awesome! That makes sense. Reworded it like that.